src/components/CaptureDetailDialog.js

```diff
@@ -115,21 +134,36 @@ function CaptureDetailDialog(props) {
     }
   }, [cdContext.capture, capture]);
 
+  async function handleCaptureTagDeletion({ capture, tag }) {
+    try {
+      await cdContext.deleteCaptureTag({
+        captureId: capture?.id,
+        tagId: tag.id,
+      });
+      if (props.onCaptureTagDelete !== undefined) {
+        props.onCaptureTagDelete();
+      }
```

Comment:
onCaptureTagDelete is showing undefined.

Reply:
It looks like `onCaptureTagDelete` is an optional callback to the calling component to make any updates it needs to.
In the case of _Verify_ there are no component updates necessary, so it makes sense that this is `undefined`.

The problem appears to be that `CaptureDetailContext` is not updating properly when a tag is deleted. It should refresh the capture detail when deletion is complete.

Reply:
Ok, I'll check it.

Reply:
Hey @mayankbucha. Did you ever manage to make any progress on completing this PR?